views/templates/hook/payment_return.tpl

```diff
@@ -29,7 +29,7 @@
 		{l s='Your check must include:' d='Modules.Checkpayment.Shop'}
 		<br /><br />- {l s='Payment amount.' d='Modules.Checkpayment.Shop'} <span class="price"><strong>{$total_to_pay}</strong></span>
 		<br /><br />- {l s='Payable to the order of' d='Modules.Checkpayment.Shop'} <strong>{if $checkName}{$checkName}{else}___________{/if}</strong>
-		<br /><br />- {l s='Mail to' d='Modules.Checkpayment.Shop'} <strong>{if $checkAddress}{$checkAddress}{else}___________{/if}</strong>
+		<br /><br />- {l s='Mail to' d='Modules.Checkpayment.Shop'} <strong>{if $checkAddress}<br />{$checkAddress nofilter}{else}___________{/if}</strong>
```

Comment:
Hmm, we never fond of not escaping the content displayed, for security reason.
Another solution coming in mind is using carriage return instead, and using nl2br methods.

Reply:
So why `nofilter` is used into [_payment_infos.tpl_ file](https://github.com/PrestaShop/ps_checkpayment/blob/master/views/templates/front/payment_infos.tpl) ? It does not seem very coherent ...

Line 34 : `<dd>{$checkAddress nofilter}</dd>`

Reply:
The `$checkAddress` value is defined by admin in Prestashop backoffice.
BTW, the `nofilter` tag is already applied to this value in others templates as indicated by @Klemart3D.